vespalib/src/tests/datastore/unique_store/unique_store_test.cpp

```diff
@@ -204,37 +214,67 @@ struct OrderedSmallOffsetNumberUniqueStore
     static constexpr DictionaryType dictionary_type = DictionaryType::BTREE;
 };
 
-struct UnorderedNumberUniqueStore
+struct HybridNumberUniqueStore
 {
     using UniqueStoreType = NumberUniqueStore;
     static constexpr DictionaryType dictionary_type = DictionaryType::BTREE_AND_HASH;
 };
 
-struct UnorderedStringUniqueStore
+struct HybridStringUniqueStore
 {
     using UniqueStoreType = StringUniqueStore;
     static constexpr DictionaryType dictionary_type = DictionaryType::BTREE_AND_HASH;
 };
 
-struct UnorderedCStringUniqueStore
+struct HybridCStringUniqueStore
 {
     using UniqueStoreType = CStringUniqueStore;
     static constexpr DictionaryType dictionary_type = DictionaryType::BTREE_AND_HASH;
 };
 
-struct UnorderedDoubleUniqueStore
+struct HybridDoubleUniqueStore
 {
     using UniqueStoreType = DoubleUniqueStore;
     static constexpr DictionaryType dictionary_type = DictionaryType::BTREE_AND_HASH;
 };
 
-struct UnorderedSmallOffsetNumberUniqueStore
+struct HybridSmallOffsetNumberUniqueStore
 {
     using UniqueStoreType = SmallOffsetNumberUniqueStore;
     static constexpr DictionaryType dictionary_type = DictionaryType::BTREE_AND_HASH;
 };
 
-using UniqueStoreTestTypes = ::testing::Types<OrderedNumberUniqueStore, OrderedStringUniqueStore, OrderedCStringUniqueStore, OrderedDoubleUniqueStore, UnorderedNumberUniqueStore, UnorderedStringUniqueStore, UnorderedCStringUniqueStore, UnorderedDoubleUniqueStore>;
+struct UnorderedNumberUniqueStore
```

Comment:
Consider renaming _Unordered_ -> _Hash_ and _Ordered_ -> _BTree_. Applies for other structs in this file as well.